src/stuff.m4

```diff
@@ -76,11 +76,13 @@ dnl The argbash script generator will pick it up and (re)generate that one as we
 dnl
 dnl $1: the filename (assuming that it is in the same directory as the script)
 dnl $2: what has been passed to DEFINE_SCRIPT_DIR as the first param
+dnl TODO: add shellcheck source directive
```

Comment:
Please expand your thought - directive of what kind? Where?

Reply:
Sorry, I meant to clean this up once I added the shellcheck source= below.  It'll be gone in a minute.